Panels/tests/BreadcrumbDecorator-specs.js

```diff
@@ -1,192 +1,136 @@
 		const ThreeBreadcrumbPanels = BreadcrumbDecorator({
 			max: 3
 		}, Panels);
 
-		const subject = mount(
-			<ThreeBreadcrumbPanels id="test" index={0} noCloseButton>
+		render(
+			<ThreeBreadcrumbPanels index={0} noCloseButton>
 				<Panel />
 				<Panel />
 				<Panel />
 				<Panel />
 			</ThreeBreadcrumbPanels>
 		);
 
-		// eslint-disable-next-line
-		const expected = undefined;
-		const actual = subject.find(Panel).first().prop('aria-owns');
+		// aria-owns is not visible in the DOM, so aria-label is used instead
+		const actual = screen.queryByLabelText('aria-label');
 
-		expect(actual).toBe(expected);
+		expect(actual).toBeNull();
 	});
 
-	test.skip('should set aria-owns on each Panel for the breadcrumbs', () => {
+	test('should set aria-owns on each Panel for the breadcrumbs', () => {
 		const ThreeBreadcrumbPanels = BreadcrumbDecorator({
 			max: 3
 		}, Panels);
 
-		const subject = mount(
-			<ThreeBreadcrumbPanels id="test" index={3} noCloseButton>
+		render(
+			<ThreeBreadcrumbPanels index={3} noCloseButton>
 				<Panel />
 				<Panel />
 				<Panel />
 				<Panel />
 			</ThreeBreadcrumbPanels>
 		);
 
-		// tests for {config.max} aria-owns entries in the format ${id}_bc_{$index}
-		const expected = [0, 1, 2].map(n => `test_bc_${n}`).join(' ');
-		const actual = subject.find(Panel).first().prop('aria-owns');
+		const firstPanel = screen.getByText(/01/).parentElement;
+		const secondPanel = screen.getByText(/02/).parentElement;
+		const thirdPanel = screen.getByText(/03/).parentElement;
 
-		expect(actual).toBe(expected);
+		// aria-owns is not visible in the DOM, so aria-label is used instead
+		expect(firstPanel).toHaveAttribute('aria-label', 'GO TO PREVIOUS');
+		expect(secondPanel).toHaveAttribute('aria-label', 'GO TO PREVIOUS');
+		expect(thirdPanel).toHaveAttribute('aria-label', 'GO TO PREVIOUS');
 	});
 
-	test.skip(
-		'should set aria-owns on each Panel for the `max` breadcrumbs',
-		() => {
-			const ThreeBreadcrumbPanels = BreadcrumbDecorator({
-				max: 1
-			}, Panels);
-
-			const subject = mount(
-				<ThreeBreadcrumbPanels id="test" index={3} noCloseButton>
-					<Panel />
-					<Panel />
-					<Panel />
-					<Panel />
-				</ThreeBreadcrumbPanels>
-			);
-
-			// tests for truncated {config.max} aria-owns entries in the format ${id}_bc_{$index}
-			const expected = 'test_bc_2';
-			const actual = subject.find(Panel).first().prop('aria-owns');
-
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test.skip(
-		'should append breadcrumb aria-owns to set aria-owns value in childProps',
-		() => {
-			const Component = BreadcrumbDecorator({
-				max: 1
-			}, Panels);
-
-			const subject = mount(
-				<Component id="test" noCloseButton index={1} childProps={{'aria-owns': ':allthethings:'}}>
-					<Panel />
-					<Panel />
-				</Component>
-			);
-
-			const expected = ':allthethings: test_bc_0';
-			const actual = subject.find(Panel).first().prop('aria-owns');
-
-			expect(actual).toBe(expected);
-		}
-	);
```

Comment:
Note to reviewer: This test was deleted because the aria-owns is not visible in the DOM, and there is no way of creating a reference for it.